mods/tnt/init.lua

```diff
@@ -384,7 +349,74 @@ minetest.register_craft({
 	}
 })
 
+function tnt.register_tnt(def)
+	local name = ""
+	if not def.name:find(':') then
+		name = "tnt:"..def.name
+	else
+		name = def.name
+		def.name = def.name:sub(def.name:find(':')+1)
+	end
+	if not def.tiles then def.tiles = {} end
```

Comment:
? According to the Lua code style guidelines the limit is 80, this line is only 45.